react/features/toolbox/components/web/Toolbox.js

```diff
@@ -1008,7 +1008,7 @@ class Toolbox extends Component<Props, State> {
                     key = 'etherpad'
                     showLabel = { true } />,
             this._shouldShowButton('videobackgroundblur')
-                && <VideoBlurButton
+                && <VideoBackgroundButton
```

Comment:
Can we make this work on safari too, minus the blur functionality?

Reply:
I need more time to investigate/debug. Maybe we can leave this for later.